package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@getsetup/embed",
-  "version": "1.7.0",
+  "version": "2.0.0",
```

Comment:
Are we considering this Major version update?

Reply:
Yeah. It probably won't affect anyone, but removing the `min-height` on the iframe is technically a breaking change. And since this package follows semver, we need to bump to version 2.